pest/src/inputs/position.rs

```diff
@@ -4,48 +4,44 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
+#[allow(unused_imports)] use std::ascii::AsciiExt;
 use std::cmp::Ordering;
 use std::fmt;
 use std::hash::{Hash, Hasher};
 use std::ops::Range;
-use std::rc::Rc;
+use std::ptr;
 
-use super::input::Input;
 use super::span;
+use super::super::util::hash_str;
 
 /// A `struct` containing a position that is tied to an `Input` which provides useful methods to
```

Comment:
All documentation mentioning `Input` should be fixed.

Reply:
:+1: